fuzzers/qemu_launcher/injection_test/README.md

```diff
@@ -0,0 +1,10 @@
+# Injection test setup
+
+To build the injection test target:
```

Comment:
Can we add these to Makefile.toml so it'll also be tested in CI?

Reply:
how would that be tested in the CI?

Reply:
ci will run the task in a fuzzer's directory as long as it has the task has the name `test`
to do this, in Makefile.toml you write

```
# Test
[tasks.test]
linux_alias = "test_unix"
mac_alias = "unsupported"
windows_alias = "unsupported"

[tasks.test_unix]
script_runner="@shell"
script='''
# do the stuff
'''
```

Reply:
I mean how would the test look like. with afl++ I could just do AFL_BENCH_JUST_ONE and then check that a crash was found.

Reply:
We usually run it in in a `timeout` and grep the logs after the run, like this
https://github.com/AFLplusplus/LibAFL/blob/925b33889f32f67a37594db4a181df9c4f182daf/fuzzers/fuzzbench/Makefile.toml#L85

Reply:
done